src/services/uploader/uploading.controller.ts

```diff
@@ -88,7 +88,7 @@ export class UploadingController {
   }
 
   @UseGuards(AuthGuard('jwt'))
-  @UseGuards(AuthGuard('jwt'), RequireHiveVerify)
+  @UseGuards(AuthGuard('jwt'))
```

Comment:
```suggestion
```

Reply:
why? It looks like this method is using protected methods? (see ln 95)

Reply:
Repeat of line 90